tensorboard/backend/application_test.py

```diff
@@ -339,5 +339,23 @@ def testExceptions(self):
       application.TensorBoardWSGIApp(logdir, plugins, multiplexer, 0)
 
 
+class DbTest(tf.test.TestCase):
+
+  def testSqliteDb(self):
+    self.addCleanup(shutil.rmtree, self.get_temp_dir())
+    db_uri = 'sqlite:' + os.path.join(self.get_temp_dir(), 'db')
+    db_module, db_connection_provider = application.get_database_info(db_uri)
```

Comment:
We're actually testing both. Right now there's no pooling so it re-opens the DB file every time you ask for a connection.